### DISCUSSION TITLE
Better UI readability proposals

### DISCUSSION THREAD

**Discussion by RaiLTC** [ppy/osu#13169] [Ideas]
Gonna be taking inspiration from CSGO which has awesome small things that make UI more readable

**Text separation between every option**:
![csgo_to9iiiu8k3](https://user-images.githubusercontent.com/35473621/97960872-e5b68a00-1dba-11eb-94e6-3f9afba1b986.png)
This makes text in options really easy to read

**Text highlight on hover**:
![csgo_Op15JBTmE3](https://user-images.githubusercontent.com/35473621/97960920-fc5ce100-1dba-11eb-9eec-5d33948820dc.png)
Helps figure out what option you're on, IMO the button alone is not enough

**Category highlight**:
https://puu.sh/GJyru/75615ae1d0.mp4
This is probably one of my favorite options so far, it helps so much to understand what options belong to what category

**Option description box**:
![csgo_4vf3PD4ScO](https://user-images.githubusercontent.com/35473621/97963738-45fbfa80-1dc0-11eb-99b6-2f3c40ed9ba5.png)

If you have other games that offer more post below, I'll add it to the list

**Comment by Game4all** [comment]
Stable already has tooltips with detailed description of settings when hovering them so it would be cool to port this over to lazer.

**Comment by Wilker-uwu** [comment]
hi! i'm not sure if this is relevant here or in another issue: i want to point out that keeping a portrait view of the UI seems nice, but it really needs some polishing yet, as the UI elements are way too small to read or click, with the buttons at the top even being smaller than the cursor circles from Android's "show touches" function.

the following screenshots were taken while the UI size was at 2x on a ridiculous resolution of 1080x2160 because Android doesn't allow me to change it.

###### note that the navigation bar obstructing the bottom part of the screen isn't really an issue, as that still hides like usual. it's just the screenshot software doing its thing.

###### (edit) another note just in case it affects anything: in Android's accessibility settings, system text size is set to "larger".

![A screenshot of the song selection screen, as shown in portrait mode.](https://user-images.githubusercontent.com/10854855/98371188-8787e200-201a-11eb-808e-aa0ae83089e9.png)

![A screenshot of the main menu screen, with the main options open, and the game settings being shown.](https://user-images.githubusercontent.com/10854855/98371987-d84c0a80-201b-11eb-9a1a-da4ea7b454a2.png)

**Comment by peppy** [comment]
This issue is already completely too general. We are aware of issues but this is not the way to report them. Locking for now.
